File: app/src/main/java/com/vsa/filmoteca/model/database/MoviesTable.java
package com.vsa.filmoteca.model.database;

import android.provider.BaseColumns;

import com.vsa.filmoteca.utils.Constants;

/**
 * Created by seldon on 16/03/15.
 */
public class MoviesTable implements BaseColumns{

    public static final String TABLE_NAME = "movies";

    public static final String COLUMN_TITLE = "title";
    public static final String COLUMN_SUBTITLE = "subtitle";
    public static final String COLUMN_DESCRIPTION = "description";
    public static final String COLUMN_DATE = "date";
    public static final String COLUMN_URL = "url";

}

File: app/src/main/java/com/vsa/filmoteca/data/repository/database/MoviesTable.kt
package com.vsa.filmoteca.data.repository.database

import android.provider.BaseColumns

/**
 * Created by seldon on 16/03/15.
 */
class MoviesTable : BaseColumns {

    companion object {
        const val TABLE_NAME = "movies"
        const val COLUMN_TITLE = "title"
        const val COLUMN_SUBTITLE = "subtitle"
        const val COLUMN_DESCRIPTION = "description"
        const val COLUMN_DATE = "date"
        const val COLUMN_URL = "url"
    }

}
